scripts/xtensa-build-all.sh

```diff
@@ -12,6 +12,7 @@ if [ "$#" -eq 0 ]
 then
 	echo "usage: xtensa-build.sh [options] platform(s)"
 	echo "       [-l] Build rimage locally"
+	echo "       [-g] Install rimage globally"
```

Comment:
I think it would be better to remove option to install it globally at all. 
Why is installing it globally bad?
1) We request root access when it can be avoided.
2) Rimage changes often and is small so it's better to build it locally on request when making fw binary. Ideally it should be dependency of `make bin` and user shouldn't care about where rimage is installed (it's how I do it in new buildsystem).

So instead of fixing global build (thanks for noticing btw), maybe it's better to just remove it completely, I guess these lines are unused now:
```bash
	if [[ "x$BUILD_LOCAL" == "x" ]]
	then
		./configure --enable-rimage
		make -j ${BUILD_JOBS}
		sudo make install
	else
```
So you can just remove them.

When someone really wants to install rimage globally (what is usually bad idea) he can do it manually/explicitly.

What do you think?

Reply:
@jajanusz 
I think remove install rimage into system is a good idea, ABI changes will always make wired issues if system rimage is not right verison. 
I know there will be kconfig to replace the configure, let's take the change to use local rimage binary first.

And how about make local build as default?

I will change the sof-doc as well.

Reply:
IMHO you can change rimage to be local by default in xtensa-build-all.sh
In autoconf/autotools please keep it as it is, because it's going to be changed anyway in new buildsystem.

Reply:
@jajanusz should this be delayed until after the new build system is merged ?